webui/apps/eidolon-ui2/app/eidolon-apps/chatbot/app_manifest.json

```diff
@@ -11,7 +11,7 @@
     "inputLabel": "How can I help you?",
     "titleOperationName": "generate_title",
     "allowSpeech": true,
-    "speechAgent": "speech-agent",
+    "speechAgent": "speech_agent",
```

Comment:
rename the agent with a -, we can't use _ in k8's names so need to start migrating

Reply:
I think the original has the hyphen

Reply:
correct. It needs the hyphen, not the _

Reply:
The file checked in has an underscore in it now